wp-content/themes/core/components/quote/Quote.php

```diff
@@ -3,31 +3,111 @@
 namespace Tribe\Project\Templates\Components;
 
 use Tribe\Project\Components\Component;
+use Tribe\Project\Templates\Components\Image as Image_Component;
+use Tribe\Project\Templates\Models\Image;
+use Tribe\Project\Theme\Config\Image_Sizes;
 
 /**
  * Class Quote
  *
- * @property string   $quote
- * @property string   $cite
+ * @property string   $quote_text
+ * @property string   $cite_name
+ * @property string   $cite_title
+ * @property string[] $cite_image
  * @property string[] $classes
- * @property string[] $quote_attrs
+ * @property string[] $attrs
+ * @property string[] $cite_classes
  * @property string[] $cite_attrs
  */
 class Quote extends Component {
 
-	public const QUOTE       = 'quote';
-	public const CITE        = 'cite';
-	public const CLASSES     = 'classes';
-	public const QUOTE_ATTRS = 'quote_attrs';
-	public const CITE_ATTRS  = 'cite_attrs';
+	private const QUOTE       = 'quote';
+	private const CITE        = 'cite';
+
+	public const QUOTE_TEXT   = 'quote_text';
+	public const CITE_NAME    = 'cite_name';
+	public const CITE_TITLE   = 'cite_title';
+	public const CITE_IMAGE   = 'cite_image';
+	public const CLASSES      = 'classes';
+	public const ATTRS        = 'attrs';
+	public const CITE_CLASSES = 'cite_classes';
+	public const CITE_ATTRS   = 'cite_attrs';
 
 	protected function defaults(): array {
 		return [
-			self::QUOTE       => '',
-			self::CITE        => '',
-			self::CLASSES     => [ 'c-quote' ],
-			self::QUOTE_ATTRS => [],
-			self::CITE_ATTRS  => [],
+			self::QUOTE        => [],
+			self::CITE         => [],
+			self::QUOTE_TEXT   => '',
+			self::CITE_NAME    => '',
+			self::CITE_TITLE   => '',
+			self::CITE_IMAGE   => [],
+			self::CLASSES      => [ 'c-quote' ],
+			self::ATTRS        => [],
+			self::CITE_CLASSES => [ 'c-quote__cite' ],
+			self::CITE_ATTRS   => [],
+		];
+	}
+
+	public function init() {
+		$this->data[ self::QUOTE ] = $this->setup_quote_text();
+		$this->data[ self::CITE ]  = $this->setup_quote_citation();
+	}
+
+	private function setup_quote_text(): array {
+		/**
+		 * Note: HTML5 validation requires that the The block's `<section>` element contain a heading.
```

Comment:
I did not know that `<section>` elements required headings. Where is this requirement coming from? An accessibility standard?

Reply:
Yeah, it's a part of the [spec for the `<section>` element](https://www.w3.org/TR/2014/PR-html5-20140916/sections.html#the-section-element):
> The theme of each section should be identified, typically by including a heading (h1-h6 element) as a child of the section element.

The HTML5 validator, and some of the more stringent a11y checkers (Site Improve, Dubbot) will flag it. In practice, my interpretation of the spec & WCAG requirements is that _any_ "sectioning" element should be identifiable. Usually by including a heading tag inside the element, but alternatives include using `aria-label` or `aria-lablledby`.

We even ran into an issue on UCI where articles with their heading tag nested several child elements deep were flagged by Site Improve as not identifiable. We had to add unique-id's to each article's heading tag and then use `aria-labelledby` on the article container to pass the audit.